python/setup.py

```diff
@@ -223,7 +223,7 @@ def long_description_contents():
 
 setup(
     name="tvm",
-    version=__version__,
+    version="0.9.0",
```

Comment:
I think we can keep `__version__` as it going to be populated with repository information?

Reply:
Not sure that’s the right approach, because once this is merged it will bypass all existing mechanisms of version resolution from git and everything will report as 0.9.0, which is not correct.

From experience, https://github.com/apache/tvm/blob/3cca6465ba685921a2f5dbe711d10f5b5ee33d33/python/tvm/_ffi/libinfo.py#L223 seems the place that needs updating.

cc @tqchen

Reply:
For a bit more context, the current mechanism will put the right version once it bumps in a version tag, and won’t show the git hash suffix.

Reply:
> Not sure that’s the right approach, because once this is merged it will bypass all existing mechanisms of version resolution from git and everything will report as 0.9.0, which is not correct.

just fyi this is to merge into the `v0.9.0` branch, not `main` so it should be the same either way (and not affect anything but release builds) but i'll push a `0.9.0` git tag instead

Reply:
Still, I feel that in case we have follow-up `0.9.x`, it won't work, so my opinion is that we should make this specific change to the literal string. I'd like @tqchen @Mousius @areusch @junrushao1994 to comment if possible, so that we can establish some guidance for future releases.

Reply:
@junrushao1994 do you know what we did before?

Reply:
what we did is that we cut the branch, and update the version number in the new 0.9 branch. The main branch cut-off ptwill update the version to 0.9.dev0 to indicate that this is the beginning of 0.9 dev cycle

Reply:
@driazati thanks for updating